Website/webb/src/common/components/BuildingDropdown.component.js

```diff
@@ -14,9 +12,15 @@ const BuildingDropdown = ({ place_holder_text, building_options, setSelectedBuil
         menuPosition={'fixed'}
         placeholder={place_holder_text}
         onChange={handleChange}
-        options={building_options} />
+        options={building_options}
+        styles={{
```

Comment:
Microsoft edge has a 3d view tool in it's developer tools that may be helpful in viewing/updating your zIndex changes. I recommend looking for other approaches for setting the Zindex dynamically/setting these values to more "reasonable" values.